config.md

```diff
@@ -415,6 +415,7 @@ Values MAY be an empty string.
 ## Extensibility
 Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown property.
 Instead they MUST ignore unknown properties.
+Properties defined for the [target platform](#platform) by the [declared version](#specification-version) of this specification MUST NOT be ignored.
```

Comment:
@tianon questioned the “for the target platform” portion of this in #819, saying [he would be surprised][1] if the runtime did *not* error when the config set a property that this spec only defined for another platform (e.g. [the Windows-only `process.user.username` in a Linux config][2]).  I, on the other hand, would be very surprised if the runtime cared about those properties at all (and I see no code in runC checking `Process.Username`).  So whatever we end up doing to clarify the intended behavior, we'll want to cover what happens in situations like that.

[1]: https://github.com/opencontainers/runtime-spec/pull/819#issuecomment-301284534
[2]: https://github.com/opencontainers/runtime-spec/pull/819#issuecomment-301247186